index.d.ts

```diff
@@ -0,0 +1,82 @@
+import { Action, StoreEnhancer, Store } from "redux";
+
+export interface StoreCreator {
+  <S, T>(
+    reducer: EnhancedReducer<S>,
```

Comment:
what are your thoughts on renaming EnhancedReducer to LoopReducer? I feel like enhanced is more generic than it needs to be.

Reply:
👍 sgtm